### PR TITLE
Provide a link to https://github.com/kubic-project/kubic-o-o in the footer for easy feedback

### PR DESCRIPTION
**Pull request by okurz**
I did not check line wrapping as I am not sure how to setup this project for
local testing. This should be mentioned in the README, e.g. a link to existing
howtos.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h6>Social Media</h6>
 
       <div class="row">
         <div class="col-md-6">
-          © 2011–2018 openSUSE contributors. License: <a href=https://www.gnu.org/licenses/fdl.txt>GNU FDL v1.3 or later</a>
+            © 2011–2018 openSUSE contributors. License: <a href=https://www.gnu.org/licenses/fdl.txt>GNU FDL v1.3 or later</a>. Visit <a href=https://github.com/kubic-project/kubic-o-o> to provide enhancements or open issues about this page.
```
Comment (mrostecki) [inline _includes/footer.html]: Why did you change indentation? Two spaces are enough.

Comment (okurz) [reply _includes/footer.html]: True, changed to 2 spaces only. The indendation does not seem to be consistent, other sections in the same file used 4 spaces
